.github/workflows/ci-rust.yml

```
@@ -77,12 +79,12 @@ jobs:
         run: |
           set -ex
           AGNOSTIC_CRATES=`python misc/libparsec_crates_flags.py agnostic`
-          cargo nextest run --cargo-profile ci-rust $AGNOSTIC_CRATES
+          cargo nextest run $CARGO_NEXTEST_CI_FLAGS $AGNOSTIC_CRATES
```

```suggestion
          cargo nextest run ${{ env.CARGO_NEXTEST_CI_FLAGS }} $AGNOSTIC_CRATES
```

use `${{ env.[...] }}`

That allow better debug experience in the CI log

---

done.

---

:ok_hand: